bots/run-queue

```diff
@@ -100,17 +105,20 @@ def main():
     with distributed_queue.DistributedQueue(opts.amqp, ['webhook', 'rhel', 'public']) as q:
         channel = q.channel
         cmd, delivery_tag = consume_webhook_queue(channel, opts.amqp)
-        if not cmd:
+        if not cmd and not delivery_tag:
             cmd, delivery_tag = consume_task_queue(channel, opts.amqp, q.declare_results['public'], q.declare_results['rhel'])
-        if not cmd:
+        if not cmd and delivery_tag:
+            logging.info("Message interpretation resulted in nop")
```

Comment:
That looks a bit weird -- we should never have an empty command from the task queue, just from the webhook queue. It feels cleaner to handle these two queues separately.

Reply:
tried something different

Reply:
Or would it be better to break off after a nop from the webhook queue? Or try to consume from the webhook queue again?

Reply:
I think it would be slightly cleaner to return after ack'ing a no-op webhook event, so that we keep treating the webhook queue as a fast and high-priority thing. However, given how many bots we have, that's minor optimization. The current code seems good enough for landing.